cmd/genericconf/config.go

```diff
@@ -109,13 +109,17 @@ func FileLoggingConfigAddOptions(prefix string, f *flag.FlagSet) {
 }
 
 type RpcConfig struct {
-	MaxBatchResponseSize int `koanf:"max-batch-response-size"`
-	BatchRequestLimit    int `koanf:"batch-request-limit"`
+	MaxBatchResponseSize             int    `koanf:"max-batch-response-size"`
+	BatchRequestLimit                int    `koanf:"batch-request-limit"`
+	EnableTeeIntegrityAttestation    bool   `koanf:"enable-tee-integrity-attestation"`
+	IntegrityKeyPairAttestationsPath string `koanf:"integrity-key-pair-attestations-path"`
 }
 
 var DefaultRpcConfig = RpcConfig{
-	MaxBatchResponseSize: 10_000_000, // 10MB
-	BatchRequestLimit:    node.DefaultConfig.BatchRequestLimit,
+	MaxBatchResponseSize:             10_000_000, // 10MB
+	BatchRequestLimit:                node.DefaultConfig.BatchRequestLimit,
+	EnableTeeIntegrityAttestation:    false,
+	IntegrityKeyPairAttestationsPath: "integrity_key_pair_attestations",
```

Comment:
Let’s add a blank line between the upstream options and our Espresso options. This helps reduce unnecessary diffs and eases the pain of syncing with upstream.